config/locales/es/admin.yml

```
@@ -1637,6 +1637,7 @@ es:
       index:
         create: Crear entidad
       new:
+        admin_information: 'Al crear una entidad se copiará su usuario actual <strong>"%{username}"</strong> con el email <strong>"%{email}"</strong> y su actual contraseña en la base de datos de la nueva entidad y se le otorgarán permisos de administración automáticamente. Nótese que serán cuentas de usuarios que están en bases de datos completamente independientes por lo que si cambia la contraseña de su usuario en una entidad las cuentas que tengas en otras entidades permanecerán intactas.'
```

I think we usually address people using "tú" instead of "usted" :thinking:.

```suggestion
        admin_information: 'Al crear una entidad se copiará tu usuario actual <strong>"%{username}"</strong> con el email <strong>"%{email}"</strong> y tu actual contraseña en la base de datos de la nueva entidad y se te otorgarán permisos de administración en la nueva entidad automáticamente. Nótese que serán cuentas de usuarios que están en bases de datos completamente independientes por lo que si cambias la contraseña de tu usuario en una entidad, las cuentas que tengas en otras entidades permanecerán intactas.'
```